x-pack/test/apm_api_integration/tests/services/agent.ts

```diff
@@ -35,12 +35,12 @@ export default function ApiTest({ getService }: FtrProviderContext) {
     () => {
       it('returns the agent name', async () => {
         const response = await supertest.get(
-          `/api/apm/services/opbeans-node/agent_name?start=${start}&end=${end}`
+          `/api/apm/services/opbeans-node/agent?start=${start}&end=${end}`
         );
 
         expect(response.status).to.be(200);
 
-        expect(response.body).to.eql({ agentName: 'nodejs' });
+        expect(response.body).to.eql({ agentName: 'nodejs', serviceRuntimeName: 'node' });
```

Comment:
nit: can we shorten this to `runtimeName`?

```suggestion
        expect(response.body).to.eql({ agentName: 'nodejs', runtimeName: 'node' });
```